demo/index.html

```diff
@@ -2,24 +2,54 @@
 <html>
 <head>
   <!--[if lt IE 9]>
-    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.2/html5shiv-printshiv.js"></script>
-    <script src="//cdnjs.cloudflare.com/ajax/libs/es5-shim/4.1.7/es5-shim.min.js"></script>
-    <script src="//cdnjs.cloudflare.com/ajax/libs/es5-shim/4.1.7/es5-sham.min.js"></script>
```

Comment:
Hm, this is just the way they are in the boilerplate (both `victory-component-boilerplate` and `formidable-react-component-boilerplate`) – we should find out why. I think I remember @ryan-roemer giving a reason.